src/pages/Home.js

```diff
@@ -62,26 +62,24 @@ export default function Home(props) {
 
   return (
     <main>
-      <h1 className="mt-5 text-3xl text-center self-start font-light">
-        Welcome to Honey Do.
+      <h1 className="mt-5 text-3xl md:text-4xl text-center self-start font-light">
+        Welcome to Honey Do
```

Comment:
It's minor for this project, but I wanted to mention that the punctuation throughout `Home.js` had been adjusted and approved with Ana's previous PR under styling https://github.com/the-collab-lab/tcl-23-smart-shopping-list/pull/68, while this current PR is addressing spacing. 

Again, minor for this PR, but given that the Figma has punctuation a certain way and (to my knowledge) there hadn't been a conversation about altering it, it would be helpful to address before implementing such changes 👍 

(Also, I apologize for any bluntness coming out of this comment! My brain is just getting a move on 🧠 )

Reply:
My two cents: I think it looks better without the punctuation :D  I say let's go with this version 🥳